packages/protocol/contracts/bridge/Bridge.sol

```diff
@@ -95,8 +95,9 @@ contract Bridge is EssentialContract, IBridge {
         external
         onlyFromNamed("bridge_watchdog")
     {
+        uint64 _timestamp = toPause ? type(uint64).max : uint64(block.timestamp);
```

Comment:
I guess have to be a bit careful here because the timestamp is used to do math on, and so if that math is done on uint64s it may overflow. I think currently it's fine because the value is upcasted to uint256s when it's used in math, and also the transactions reverting with overflow is okay for these messages.